10300.xml

```diff
@@ -0,0 +1,142 @@
+
+For more information about this object, please contact Bilhanan.Silverajan@tuni.fi
+
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>LWM2M Meta Object</Name>
+    <Description1>
+	  <![CDATA[
+	  Use this meta object to provide one or more locations hosting XML data
+	  model definitions for reusable or private and vendor-specific devices,
+	  such as those in the range 26241-42768.
+	  ]]>
+	</Description1>
+    <ObjectID>10300</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10300</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="0">
+        <Name>ObjectID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description>
+		  <![CDATA[
+		  Object ID of the vendor-specific object. If a finer granularity is
+		  needed to identify the object (for example, Object IDs in the range
+		  26241 - 32768), the optional ObjectInstanceHandle resource can also be
+		  used.
+		  ]]>
+		</Description>
+      </Item>
+      <Item ID="1">
+        <Name>ObjectURN</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description>
+		  <![CDATA[
+		  Object URN of the vendor-specific object. URN-equivalence resolution
+		  must conform to Section 3 of RFC 8141.
+		  ]]>
+		</Description>
+      </Item>
+      <Item ID="4000">
+        <Name>ObjectInstanceHandle</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Objlnk</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units></Units>
+        <Description>
+          <![CDATA[The object link is used to refer an Instance of a given Object. An Object link value is composed of two concatenated 16-bits unsigned integers following the Network Byte Order convention. The Most Significant Halfword is an ObjectID, the Least Significant Hafword is an ObjectInstance ID.An Object Link referencing no Object Instance will contain the concatenation of 2 MAX-ID values (null link. Test 5]]>
```

Comment:
Seems that we need to fix the description of this RR
>(null link. Test 5

Reply:
also in the description, the word ```Hafword``` should be updated ```Halfword```

Reply:
The description was copied ad-verbatim from line 21 of Common.xml, which is the description of RR 4000. If I need to take action and modify the description for ObjectID 10300, please let me know. 

As I understand, two amendments are needed: "Hafword" should be corrected to "Halfword", and "(null link. Test 5" should be omitted completely.

Reply:
This update will be reviewed and updated by the Working Group